HomeAssistant.xcodeproj/project.pbxproj

```diff
@@ -27,6 +27,14 @@
 		1141182A24AFA10900E6525C /* WebhookResponseHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1141182924AFA10900E6525C /* WebhookResponseHandler.swift */; };
 		1141182B24AFA10900E6525C /* WebhookResponseHandler.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1141182924AFA10900E6525C /* WebhookResponseHandler.swift */; };
 		114FACAE24B2ABA2006C581F /* Promise+WebhookJson.test.swift in Sources */ = {isa = PBXBuildFile; fileRef = 114FACAD24B2ABA2006C581F /* Promise+WebhookJson.test.swift */; };
+		117166C124B8064D00EF1043 /* WebSocketAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C024B8064D00EF1043 /* WebSocketAPI.swift */; };
+		117166C224B8064D00EF1043 /* WebSocketAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C024B8064D00EF1043 /* WebSocketAPI.swift */; };
+		117166C424B8242E00EF1043 /* WebSocketData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C324B8242E00EF1043 /* WebSocketData.swift */; };
+		117166C524B8242E00EF1043 /* WebSocketData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C324B8242E00EF1043 /* WebSocketData.swift */; };
+		117166C724B8244700EF1043 /* WebSocketRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C624B8244700EF1043 /* WebSocketRequest.swift */; };
+		117166C824B8244700EF1043 /* WebSocketRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C624B8244700EF1043 /* WebSocketRequest.swift */; };
+		117166CA24B8246B00EF1043 /* WebSocketEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C924B8246B00EF1043 /* WebSocketEvent.swift */; };
+		117166CB24B8246B00EF1043 /* WebSocketEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = 117166C924B8246B00EF1043 /* WebSocketEvent.swift */; };
```

Comment:
Why all the dupes?

Reply:
Not sure,  but it does it for other project files in  other  folders,  too. Perhaps  it's a file reference and  a reference to its group location.